File: fizzbuzz.fs
let fizzbuzz n =
    match n % 3, n % 5 with
    | 0, 0 -> "FizzBuzz"
    | 0, _ -> "Fizz"
    | _, 0 -> "Buzz"
    | _ -> string n

[<EntryPoint>]
let main argv =
    for i in 1 .. 100 do
        printfn "%s" (fizzbuzz i)
    0

File: fizzbuzz.R
fizzbuzz <- function(n) {
  if (n %% 15 == 0) {
    "FizzBuzz"
  } else if (n %% 3 == 0) {
    "Fizz"
  } else if (n %% 5 == 0) {
    "Buzz"
  } else {
    as.character(n)
  }
}

for (i in 1:100) {
  print(fizzbuzz(i))
}
